src/core/function_lollipop.tf

```
@@ -0,0 +1,255 @@
+
+      // Keepalive fields are all optionals
+      FETCH_KEEPALIVE_ENABLED             = "true"
+      FETCH_KEEPALIVE_SOCKET_ACTIVE_TTL   = "110000"
+      FETCH_KEEPALIVE_MAX_SOCKETS         = "40"
+      FETCH_KEEPALIVE_MAX_FREE_SOCKETS    = "10"
+      FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
+      FETCH_KEEPALIVE_TIMEOUT             = "60000"
+
+      COSMOSDB_NAME                = "db"
+      COSMOSDB_URI                 = data.azurerm_cosmosdb_account.cosmos_api.endpoint
+      COSMOSDB_KEY                 = data.azurerm_cosmosdb_account.cosmos_api.primary_key
+      COSMOS_API_CONNECTION_STRING = format("AccountEndpoint=%s;AccountKey=%s;", data.azurerm_cosmosdb_account.cosmos_api.endpoint, data.azurerm_cosmosdb_account.cosmos_api.primary_key)
+    }
+  }
+}
+
+resource "azurerm_resource_group" "lollipop_rg" {
+  name     = format("%s-lollipop-rg", local.project)
+  location = var.location
+
+  tags = var.tags
+}
+
+# Subnet to host admin function
+module "lollipop_snet" {
+  source                                    = "git::https://github.com/pagopa/terraform-azurerm-v3.git//subnet?ref=v4.1.15"
+  name                                      = format("%s-lollipop-snet", local.project)
+  address_prefixes                          = var.cidr_subnet_fnlollipop
+  resource_group_name                       = data.azurerm_resource_group.vnet_common_rg.name
+  virtual_network_name                      = data.azurerm_virtual_network.vnet_common.name
+  private_endpoint_network_policies_enabled = false
+
+  service_endpoints = [
+    "Microsoft.Web",
+    "Microsoft.AzureCosmosDB",
+    "Microsoft.Storage",
+  ]
+
+  delegation = {
+    name = "default"
+    service_delegation = {
+      name    = "Microsoft.Web/serverFarms"
+      actions = ["Microsoft.Network/virtualNetworks/subnets/action"]
+    }
+  }
+}
+
+module "function_lollipop" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//function_app?ref=v4.1.15"
+
+  resource_group_name = azurerm_resource_group.lollipop_rg.name
+  name                = format("%s-lollipop-fn", local.project)
+  location            = var.location
+  domain              = "IO-COMMONS"
+  health_check_path   = "/info"
+
+  os_type          = "linux"
+  linux_fx_version = "NODE|16"
+  runtime_version  = "~4"
+
+  always_on                                = "true"
+  application_insights_instrumentation_key = data.azurerm_application_insights.application_insights.instrumentation_key
+
+  app_service_plan_info = {
+    kind                         = var.function_lollipop_kind
+    sku_tier                     = var.function_lollipop_sku_tier
+    sku_size                     = var.function_lollipop_sku_size
+    maximum_elastic_worker_count = 0
+  }
+
+  app_settings = merge(
+    local.function_lollipop.app_settings,
+  )
+
+  internal_storage = {
+    "enable"                     = true,
+    "private_endpoint_subnet_id" = data.azurerm_subnet.private_endpoints_subnet.id,
```

Code review failing. I think this will fix
```suggestion
    "private_endpoint_subnet_id" = module.private_endpoints_subnet.id,
```

---

Fixed in [8f85fc6](https://github.com/pagopa/io-infra/pull/429/commits/8f85fc65ffa189cf69246a35d91d7f6b625657ea)